src/i18n/en.toml

```diff
@@ -291,6 +293,18 @@ buttonFacebook = "Login via Facebook"
 title = "Log your rides from your mobile phone!"
 description = "Download the app and log rides from anywhere."
 
+[onboarding]
+buttonInviteFriends = "Invite friends"
+descriptionStep1 = "You will contribute to your health and mental well-being, cleaner air, a cleaner planet, healthier and safer cities. How does it work? Watch the video below:"
+descriptionInvite = "Message your colleagues, coworkers, friends, or loved ones to join you in the challenge!"
+labelInviteEmailAddresses = "Email addresses, comma separated (optional)"
```

Comment:
Text is incorrect _(optional)_, because validation rules require value (one email address) https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/324/files#diff-460c7f93bb004e11f303245df5d8dcf76819445820761867d05ac130c0e776a5R116.

Reply:
Agree. Removed.